### PR TITLE
Fix some common criticals

### PR DESCRIPTION
**Pull request by oscfdezdz**

### CODE REVIEW

File: src/exm-types.h [hunk 1]
```diff
@@ -17,6 +17,8 @@ typedef enum
     EXM_EXTENSION_STATE_OUT_OF_DATE = 4,
     EXM_EXTENSION_STATE_DOWNLOADING = 5,
     EXM_EXTENSION_STATE_INITIALIZED = 6,
+    EXM_EXTENSION_STATE_DISABLING = 7,
+    EXM_EXTENSION_STATE_ENABLING = 8,
```
Comment (oscfdezdz) [inline src/exm-types.h:21]: Not sure if this is the proper way to fix them.

Comment (mjakeman) [reply src/exm-types.h:21]: I think this is fine - I've seen this be done in other GLib projects so happy with it

**Review by mjakeman — APPROVED** [review]
Nice!

I should probably fix some of the logging done by `text-engine` so we only have essential logging 🤔

**Comment by mjakeman** [comment]
@oscfdezdz Could you fix the conflicts please? Then happy to merge

**Comment by oscfdezdz** [comment]
> @oscfdezdz Could you fix the conflicts please? Then happy to merge

Done!

**Pull request merged**